engine/swarm/services.md

```diff
@@ -70,6 +70,86 @@ $ docker service create --name helloworld alpine ping docker.com
 9uk4639qpg7npwf3fn2aasksr
 ```
 
+### Specify the image version the service should use
+
+When you create a service without specifying any details about the version of
+the image to use, the service uses the latest version available in Docker Hub
+or your local registry. You can force the service to use a specific version of
+the image in a few different ways, depending on your desired outcome.
+
+When you don't specify a tag, you get the latest version of an image, because
```

Comment:
nit: `latest` tag rather than latest version.

`latest` doesn't have any special properties, so it's possible for it to be out of date. It's only the latest version by convention.